test/distributions/test_multivariate_normal.py

```diff
@@ -186,6 +201,47 @@ def test_multivariate_normal_batch_correlated_sampels_cuda(self):
         if torch.cuda.is_available():
             self.test_multivariate_normal_batch_correlated_sampels(cuda=True)
 
+    def test_log_prob(self):
```

Comment:
Can you actually test this against the log_prob call on `torch.distributions.MultivariateNormal` to ensure that this remains fully compatible with that?

Reply:
Just pushed a commit to do this